pkg/madmin/README.md

```diff
@@ -262,6 +262,17 @@ Fetches Mem utilization for all cluster nodes. Returned value is in Bytes.
 | `mem.Usage.Mem`   | _uint64_ | The total number of bytes obtained from the OS         |
 | `mem.Usage.Error` | _string_ | Error (if any) encountered while accesing the CPU info |
 
+<a name="NetPerfInfo"></a>
+### NetPerfInfo() (map[string][]NetPerfInfo, error)
```

Comment:
A line describing the `string` key in the returned `map[string]NetPerInfo` would be useful for the users.

Reply:
Done